hoprd/rest-api/src/account.rs

```diff
@@ -188,6 +190,10 @@ pub(super) async fn withdraw(
             }),
         )
             .into_response(),
+        Err(HoprLibError::StatusError(HoprStatusError::NotRunningError)) => {
+            (StatusCode::PRECONDITION_FAILED, ApiErrorStatus::NotReady).into_response()
```

Comment:
What status code is this? And is it being used in the schema doc?

Reply:
412,  in line with the responses that the endpoint can return.